contracts/cw721-base/README.md

```diff
@@ -30,6 +30,39 @@ or just create a custom contract as the owner and use that contract to Mint.
 
 If provided, it is expected that the _token_uri_ points to a JSON file following the [ERC721 Metadata JSON Schema](https://eips.ethereum.org/EIPS/eip-721).
 
+## Updating the minter
```

Comment:
I think this is better put in a separate `MIGRATION.md` which contains all the migration guides ordered by versions

Reply:
Like this: https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md

Reply:
great idea, thanks